MIT Press.js

```diff
@@ -0,0 +1,206 @@
+{
+	"translatorID": "ac277fbe-000c-46da-b145-fbe799d17eda",
+	"label": "MIT Press",
+	"creator": "Guy Aglionby",
+	"target": "https://(www\\.)?mitpress\\.mit\\.edu/",
+	"minVersion": "3.0",
+	"maxVersion": "",
+	"priority": 100,
+	"inRepository": true,
+	"translatorType": 4,
+	"browserSupport": "gcsibv",
+	"lastUpdated": "2019-10-27 22:08:32"
+}
+
+/*
+	***** BEGIN LICENSE BLOCK *****
+
+	Copyright © 2019 Guy Aglionby
+	This file is part of Zotero.
+
+	Zotero is free software: you can redistribute it and/or modify
+	it under the terms of the GNU Affero General Public License as published by
+	the Free Software Foundation, either version 3 of the License, or
+	(at your option) any later version.
+
+	Zotero is distributed in the hope that it will be useful,
+	but WITHOUT ANY WARRANTY; without even the implied warranty of
+	MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+	GNU Affero General Public License for more details.
+
+	You should have received a copy of the GNU Affero General Public License
+	along with Zotero.  If not, see <http://www.gnu.org/licenses/>.
+
+	***** END LICENSE BLOCK *****
+*/
+
+function detectWeb(doc, url) {
+	if (url.includes('/books/') && !url.includes('/series/')) {
+		return 'book';
+	}
+	else if ((url.includes('/mit-press-open') || url.includes('/contributors/')
+		|| url.includes('/search?') || url.includes('/series/')) && getSearchResults(doc, true)) {
+		return 'multiple';
+	}
+	return false;
+}
+
+function doWeb(doc, url) {
+	if (detectWeb(doc, url) === 'multiple') {
+		Zotero.selectItems(getSearchResults(doc, false), function (selected) {
+			if (selected) {
+				ZU.processDocuments(Object.keys(selected), scrape);
+			}
+		});
+	}
+	else {
+		scrape(doc);
+	}
+}
+
+function scrape(doc, url) {
+	let authorsElement = ZU.xpath(doc, '//span[@class = "book__authors"]/p[1]/a');
+	let authors = authorsElement.map(function (author) {
```

Comment:
Could we try to distinguish editors by using the "Edited by" wording, e.g. https://mitpress.mit.edu/books/construction-site-possible-worlds

Reply:
Example of book with editor and author: https://mitpress.mit.edu/books/kings-and-things
Example of book with author and translator: https://mitpress.mit.edu/books/ribbon-olympias-throat